Cargo.toml

```diff
@@ -17,7 +17,7 @@ clap = { version = "4.0.17", features = ["cargo"] }
 colored = "2.0.0"
 flate2 = "1.0"
 infer = "0.15.0"
-ion-rs = { version = "1.0.0-rc.5", features = ["experimental"] }
+ion-rs = { version = "1.0.0-rc.6", git = "https://github.com/amazon-ion/ion-rust.git", branch = "version-bump-rc6", features = ["experimental"] }
```

Comment:
🗺️ This PR depends on amazon-ion/ion-rust#785. Once that's merged/released, I'll update this to point to crates.io.